source/ota.c

```diff
@@ -305,42 +305,47 @@ static OtaStateTableEntry_t otaTransitionTable[] =
     { OtaAgentStateAll,                 OtaAgentEventShutdown,            shutdownHandler,        OtaAgentStateStopped             },
 };
 
+#ifndef LogInfo
```

Comment:
Did you mean ifdef? and why LogInfo?

Reply:
Thanks for the callout, suppressed these warnings in OTA_Init()